vignettes/FDZ_Data_Cleaning.Rmd

```diff
@@ -47,6 +48,15 @@ The data set is loaded via the `import_spss()` command. The data set contains 14
 Note that the data set is imported as a `GADSdat` object. For detailed explanations on the corresponding object structure and technical details, we refer readers to the `eatGADS` [documentation](https://beckerbenj.github.io/eatGADS/articles/import_spss.html). 
 
 
+## Removing all non-ASCII characters
+Some systems are designed to handle only ASCII characters. If your data contains non-ASCII characters, it might cause compatibility issues or errors when processing the data. `fixEncoding()` removes special characters from a character vector or a `GADSdat` object
```

Comment:
Nits: 
Maybe substitute the first sentence with:
`Non-ASCII characters frequently lead to issues depending on the operating system and statistical software used.`
and consistently used `non-ASCII` characters instead of `special characters` (I am aware that I use the term throughout `eatGADS` and `eatFDZ` documentation as well, but I think this is incorrect.)

Reply:
like the first sentence a lot + consistency with "non-ASCII characters"